core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -232,20 +236,22 @@ public String getEnvVarOrUserProperty(@NotNull final String propertyName, @Nulla
     /**
      * Gets a configured setting from a the user's configuration file.
      * The configuration file will be the <code>.testcontainers.properties</code> file in the user's home directory.
+     * <p>
+     * Note that only environment variables prefixed <code>TESTCONTAINERS_</code> will be searched.
      *
      * @param propertyName name of configuration file property (dot-separated lower case)
      * @return the found value, or null if not set
      */
     @Contract("_, !null -> !null")
-    public String getUserProperty(@NotNull final String propertyName, @Nullable final String defaultValue) {
+    public String getEnvVar(@NotNull final String propertyName, @Nullable final String defaultValue) {
```

Comment:
Have renamed this method, as it was misleadingly named. It is actually unused, so we might want to consider removal.

Reply:
the method is public, we would need to deprecate it first (also, japicmp should bark on this change :D)

Reply:
Ah yes, I'll deprecate and revert.

Reply:
Ah I'm a fool: this method may be unused but is correctly named - it will read settings from `TESTCONTAINERS_` prefixed env vars.

The javadoc is wrong, so I'll fix it.